lib/interactive.py

```diff
@@ -223,52 +221,38 @@ def plot_route(self, messageId):
 
-                    if "hopLimit" in p.packet:
-                        hopLimit = p.packet["hopLimit"]
-                    else:
-                        hopLimit = None
-
                     if p.packet["from"] == tx.hwId:
                         if "requestId" in p.packet["decoded"]:
                             if p.packet["priority"] == "ACK":
-                                msgType = "Real ACK"
+                                msgType = "Real\/ACK"
                             else:
                                 msgType = "Response"
                         else:
                             msgType = "Original message"
                     elif "requestId" in p.packet["decoded"]:
                         if p.packet["decoded"]["simulator"]["portnum"] == "ROUTING_APP":
-                            msgType = "Forwarding real ACK"
+                            msgType = "Forwarding\/real\/ACK"
                         else:
-                            msgType = "Forwarding response"
+                            msgType = "Forwarding\/response"
                     else:
                         if int(p.packet['from']) == rx.hwId:
-                            msgType = "Implicit ACK"
+                            msgType = "Implicit\/ACK"
                         else:
                             if to == "All":
                                 msgType = "Rebroadcast"
                             else:
-                                msgType = "Forwarding message"
-
-                    fields = []
-                    msgTypeField = r"$\bf{"+msgType+"}$"
-                    fields.append(msgTypeField)
-                    origSenderField = "Original sender: "+str(p.packet["from"]-HW_ID_OFFSET)
-                    fields.append(origSenderField)
-                    destField = "Destination: "+to
-                    fields.append(destField)
-                    portNumField = "Portnum: "+str(p.packet["decoded"]["simulator"]["portnum"])
-                    fields.append(portNumField)
-                    if hopLimit:
-                        hopLimitField = "HopLimit: "+str(hopLimit)
-                        fields.append(hopLimitField)
-                    rssiField = "RSSI: "+str(round(p.rssis[ri], 2)) + " dBm"
-                    fields.append(rssiField)
-                    table = ""
-                    for i, f in enumerate(fields):
-                        table += f
-                        if i != len(fields)-1:
-                            table += "\n"
+                                msgType = "Forwarding\/message"
+
+                    hopLimit = p.packet.get("hopLimit")
+
+                    fields = [ r"$\bf{" + msgType + "}$"
+                             , f"Original sender: {p.packet['from'] - HW_ID_OFFSET}"
+                             , f"Destination: {to}"
+                             , f"Portnum: {p.packet["decoded"]["simulator"]["portnum"]}"
```

Comment:
I'm now still getting a syntax error for the same reason here. 

Maybe it depends on the Python version? I'm using 3.10.12.

Reply:
@r41d Any update on this? I meant that I now get the error on line 251, so I think it just needs the same treatment as line 249, with the single quotes.

Reply:
@GUVWAF I was on vacation and am planning to look into it real soon